app/views/sections/guarantor/GuarantorArrangerView.scala.html

```diff
@@ -0,0 +1,54 @@
+@*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import models.requests.DataRequest
+@import models.sections.guarantor.GuarantorArranger
+
+@this(
+    layout: templates.Layout,
+    formHelper: FormWithCSRF,
+    govukErrorSummary: GovukErrorSummary,
+    govukRadios: GovukRadios,
+    govukButton: GovukButton,
+    h1: components.h1,
+    continueOrExit: components.continueOrExit
+)
+
+@(form: Form[_], mode: Mode)(implicit request: DataRequest[_], messages: Messages)
+
+@layout(pageTitle = title(form, messages("guarantorArranger.title"))) {
+
+    @formHelper(action = controllers.sections.guarantor.routes.GuarantorArrangerController.onSubmit(request.ern, request.lrn, mode), 'autoComplete -> "off") {
+
+        @if(form.errors.nonEmpty) {
+            @govukErrorSummary(ErrorSummaryViewModel(form,  errorLinkOverrides = Map("value" -> "value_0")))
+        }
+
+        @govukRadios(
+            RadiosViewModel(
+                field  = form("value"),
+                legend = LegendViewModel(HtmlContent(heading)).withCssClass("govuk-fieldset__legend govuk-fieldset__legend--l"),
+                items  = GuarantorArranger.options
+            )
+        )
+
+        @continueOrExit()
+    }
+}
+
+@heading = {
+    @h1("guarantorArranger.heading", Some(messages("guarantor.subHeading", request.lrn)), "govuk-fieldset__heading")
+}
```

Comment:
```suggestion
}

@{
    //$COVERAGE-OFF$
}
```

Just to help coverage along a little (means that we're not penalised by Play's auto-generated apply methods)

Reply:
Updated